reinforcement_learning/environment/dialogue_env_moviebot.py

```diff
@@ -354,10 +355,3 @@ def _save_dialogue_history(self) -> None:
 
         with open(file_name, "w", encoding="utf-8") as outfile:
             json.dump(json_file, outfile, indent=4)
```

Comment:
Why was `gym.register` removed?

Reply:
I moved it to `__init__.py` to automate the registration step.